src/js/index.js

```diff
@@ -8,22 +8,43 @@ import 'srcdoc-polyfill';
 
 import '../styles/clsp-player.scss';
 
-import IovCollection from './iov/IovCollection';
 import Iov from './iov/Iov';
+import IovCollection from './iov/IovCollection';
 import TourController from './iov/TourController';
 import utils from './utils/utils';
 
-window.IovCollection = IovCollection;
-window.TourController = TourController;
-const ClspUtils = utils;
+function register () {
+  // @todo - deprecate `window` mutations.  use `globalThis`
+  if (!window.IovCollection) {
+    window.IovCollection = IovCollection;
+  }
+
+  if (!window.TourController) {
+    window.TourController = TourController;
+  }
 
-if (!window.clspUtils) {
-  window.clspUtils = utils;
+  if (!window.clspUtils) {
+    window.clspUtils = utils;
+  }
 }
 
+// @todo - do not mutate `window`, the caller should do it as needed
```

Comment:
i don't think we should provide a way to de-register, and sure, we can do it now.  it just seemed out of scope.  now is a good a time as any though, since we only have a single client using this version, and they are expecting changes, AND we're modifying the import procedure.

Reply:
per discussion let's do it

Reply:
further discussion is needed for this

Reply:
actually i found what i need - i don't know where it's documented, but webpack will package the application to either attach itself to window or not based on the environment.  Therefore, it will be attached to window by default when importing via a script tag, and not attached to window when imported or required

Reply:
done - i had to change the variables names (e.g. utils) to be namespaced for/with Clsp